Lab1.ipynb

```diff
@@ -29,13 +29,20 @@
    "id": "835e4211-6810-4209-b46d-69668f342916",
    "metadata": {},
    "source": [
+    "#### A.\n",
+    "The normal distribution is a symmetrically distributed bell curve of the outcomes or potential outcomes for a given dataset. The probability density of the normal distribution is given by:\n",
+    "\n",
+    "$$P(x) = \\frac{1}{\\sigma\\sqrt{2pi}} e^{-(x-\\mu)^2/({2\\sigma^2})}$$\n",
+    "\n",
+    "where x=value, $\\mu$ = mean (the center of the distribution), and $\\sigma^2$ is the variance of the distribution. For the standard normal distribution (where mean=0 and sigma=1), the Z-score is the value for which a given percentage of the data is above that value. \n",
+    "\n",
     "#### B.\n",
-    "erfc(1) tells us that 15.7% of distribution is above Z score of 1 while erf(1) tells us 84.2% of data is below Z score of 1. "
+    "erfc(1) tells us that 15.7% of distribution is above Z score of 1 while erf(1) tells us 84.3% of data is below Z score of 1. These values match up with those from the Z table. I also show that erfc(1)+erf(1) = 1, to demonstrate that the integral of the normal distribution across all space is 1. I have some confusion why erf(1) and cdf(1) are not equivalent in the 1/100th place, as these should calculate the same quantity."
```

Comment:
Here you explained the erfc() function well, but you could also explain cdf() as well to fully flesh out your explanation of your code here.